scheduler/test/cook/test/mesos/util.clj

```diff
@@ -63,6 +63,24 @@
     ;; u3 has no jobs (running or otherwise)
     (is (= 0 (count (util/get-user-running-job-ents (db conn) "u3"))))))
 
+(deftest test-cache
```

Comment:
Should we also test for eviction of entries in the cache?

Reply:
I'm willing to say 'trust guava'? :)

Reply:
It basically ends up testing whether we have invoked guava correctly to configure the eviction policy :)

Reply:
Yup.